cypress/integration/Platform/index.js

```diff
@@ -3,37 +3,40 @@ Given('you open the App without auth cookie', () => {
 });
 
 Then('you should see the login prompt', () => {
-    cy.get('[data-test="dhis2-adapter-loginsubmit"]')
-        .should('exist');
+    cy.get('[data-test="dhis2-adapter-loginsubmit"]');
+
+    cy.get('[data-test="dhis2-adapter-loginname"]')
+        .find('#j_username');
+
+    cy.get('[data-test="dhis2-adapter-loginpassword"]')
+        .find('#j_password');
+
+    cy.get('#j_username');
+
+    cy.get('#j_password');
 });
 
 When('you fill in credentials', () => {
     cy.get('[data-test="dhis2-adapter-loginname"]')
         .find('input')
         .type('admin');
+
     cy.get('[data-test="dhis2-adapter-loginpassword"]')
         .find('input')
         .type('district');
 });
 
 When('you sign in', () => {
-    cy.get('[data-test="dhis2-adapter-loginsubmit"]')
-        .click();
+    cy.get('form')
```

Comment:
what is the `form` here, is it id?

Reply:
The login inputs are wrapped in a form DOM element. The form DOM element is what we're getting here.

Reply:
Thanks for the explanation @JoakimSM makes sense